src/routes/Profile/FreeTimeActivities/FreeTimeActivities.tsx

```diff
@@ -29,7 +28,8 @@ const FreeTimeActivities = () => {
   const actionBar = useActionBar();
   const [isOpen, setIsOpen] = React.useState(false);
   const [rows, setRows] = React.useState(getFreeTimeActivitiesTableRows(vapaaAjanToiminnat));
-  const { csrf } = useAuth() as { csrf: NonNullable<RootLoaderData['csrf']> };
+  const auth = useAuth();
+  const csrf = auth!.csrf;
```

Comment:
Maybe add some info about the change as it is changed in many places regarding this auth. And it does not seem clearly specific for the purpose of the PR at hand.